app/views/unattended/provisioning_templates/snippet/redhat_kickstart_rhsm.erb

```diff
@@ -0,0 +1,49 @@
+<%- if host_param('syspurpose_role') || host_param('syspurpose_usage') || host_param('syspurpose_sla') || host_param('syspurpose_addons') %>
```

Comment:
This is a left over. Should be removed. However, there is a note there that we should also parse syspurpose_addons. Wasn't in the example docs, looks to be in the anaconda source.